### DISCUSSION TITLE
www.virustotal.com shows TROJ_GEN.R002V01J123 in one result for LocalSend-1.11.1-windows-x86-64.msix

### DISCUSSION THREAD

**Discussion by sergeen** [localsend/localsend#756] [Discussion (English, International)]
I usually scan installers using [www.virustotal.com](http://www.virustotal.com/), and one vendor, TrendMicro-HouseCall, has shown an alert for what they believe to be malicious code. You can find more information about this alert on [this link](https://www.virustotal.com/gui/file/a917384791e1bd17fbe33e0c54622cbb1a00501581a3d103fb8461ba36143c8d).

I don't believe this is a real threat, but I wanted to share it to be sure. Is anyone else experiencing this issue?

**Comment by Tienisto** [comment]
This is weird, maybe because this app starts a server and is able to write data to the hard drive which might be suspicious.

**Reply by sergeen** [reply]
Sadly, the website does not provide comprehensive information about the technique used to flag the file.
